user-interface/src/main/resources/impressum/LegalNotice.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<body>
+<div>
+  <h1>Legal Notice</h1>
+  <h2 id="m46">Provider</h2>
+  <p>University of Tübingen</p>
+  <p>Quantitative Biology Center, M3 Research Center</p>
+  <p>Otfried-Müller-Straße 37</p>
+  <p>72076 Tübingen</p>
+  <p>Germany</p>
+
+  <h2 id="m56">Contact Options</h2>
+  <p>E-Mail Address: sven.nahnsen@uni-tuebingen.de</p>
+  <p>Phone: +49-7071-29-82534</p>
+
+  <h2 id="m153">Representatives</h2>
+  <p>Representative/s: Prof. Dr. Sven Nahnsen</p>
+
+  <h2 id="m58">Company Details</h2>
+  <p>VAT identification number: DE812383453</p>
+
+  <h2 id="m169">Image credits</h2>
+  <p>Image Sources and Copyright Notices: </p>
+  <p>Mouse by Graphic Mall on IconScout</p>
+  <p>Mushroom by Jemis Mali on IconScout</p>
+  <p>Kidneys by Daniel Burka on IconScout</p>
+  <p>Liver by Daniel Burka on IconScout</p>
+  <p>Heart by Vector Stall on IconScout</p>
+  <p>Leaf by Phosphor Icons on IconScout</p>
```

Comment:
Please provided the actual URLs to the content and get in contact with @wow-such-code about the right attribution formatting.

Reply:
This is done!